data/transaction/interface.go

```diff
@@ -11,3 +17,15 @@ type Marshalizer interface {
 	Marshal(obj interface{}) ([]byte, error)
 	IsInterfaceNil() bool
 }
+
+type StatusComputerHandler interface {
```

Comment:
Missing comment on exported item (please set your IDE to point out this)

Reply:
Done!